lib/web/mage/backend/floating-header.js

```diff
@@ -48,6 +48,7 @@ define([
             this.element.wrapInner($('<div/>', {
                 'class': 'page-actions-inner', 'data-title': title
             }));
+            this.element.removeClass('floating-header');
```

Comment:
I'm not sure if this workaround is necessary.
Is there some specific purpose?

I guess, if removing the `floating-header` class resolves some minor issues when Page Actions Bar is fixed to the top, then we can try to fix these issues by CSS instead.
e.g. by adding `width: 100%` for `.page-actions-inner` block inside of the `.floating-header._fixed` etc.

Reply:
Well. The common reason why issue appears, is missing of display: flex and justify-content: flex-end of ".page-actions-buttons" that does not in .page-actions. That's why I added .floating-header that extends all properties of .page-actions-buttons to simulate already initialized floatingHeader widget with flexbox properties. Its ok now with this class even if we keep it after widget initializion. However its possible that some other properties will be added and can break header.

Reply:
.page-actions-inner are not during page rendering. It appears only after jquery widget are initialized

Reply:
> Its ok now with this class even if we keep it after widget initializion. However its possible that some other properties will be added and can break header.
It does make sense. Let's stick with the current solution.